### PR TITLE
feat(Docs): Allow basic auth to be used for journey recalc API (LLC-614)

### PR DESCRIPTION
**Pull request by PrinceWaune**
Closes LLC-[614](https://learningpool.atlassian.net/browse/LLC-614)

### CODE REVIEW

File: src/http-journeys.md [hunk 1]
```diff
@@ -765,3 +765,152 @@ remainingCount | Number | Total amount of statements left to process
...
+    "journey": {
+        "recalculateStatus": {
+            "pushing": false,
+            "inProgress": false,
+            "totalCount": 3,
+            "remainingCount": 0,
+            "recalculatedAt": "2020-12-14T10:29:22.439Z",
+            "cancelledAt": "2020-12-14T10:42:09.921Z",
+            "cancelledBy": "5fd0d4de5fa8774c995b2258",
+            "recalculatedBy": "5fd0d4de5fa8774c995b2258",
```
Comment (lucuilearningpool) [inline src/http-journeys.md:867]: These seem to be the old attributes, please confirm if they are still needed @PrinceWaune?

File: src/http-journeys.md [hunk 2]
```diff
@@ -765,3 +765,152 @@ remainingCount | Number | Total amount of statements left to process
...
+    "journey": {
+        "recalculateStatus": {
+            "pushing": true,
+            "inProgress": true,
+            "totalCount": 0,
+            "remainingCount": 0,
+            "recalculatedAt": "2020-12-14T10:29:22.439Z",
+            "cancelledAt": "2020-12-10T09:53:45.929Z",
+            "cancelledBy": "5fd0d4de5fa8774c995b2258",
+            "recalculatedBy": "5fd0d4de5fa8774c995b2258",
```
Comment (lucuilearningpool) [inline src/http-journeys.md:794]: These seem to be the old attributes, please confirm if they are still needed @PrinceWaune ?

**Review by lucuilearningpool — COMMENTED** [review]
General questions around use of old attribute names - please see comments.

File: src/http-journeys.md (hunk 1 above)
Comment (PrinceWaune) [reply src/http-journeys.md:867]: Oh, yeah, nice spot @lucuilearningpool . Thanks👌

**Review by lucuilearningpool — APPROVED** [review]
Reviewed with @nicofrancisco

**Pull request merged**